test/container/2.0.0/Dockerfile.dlc.cpu

```diff
@@ -0,0 +1,8 @@
+ARG region
+FROM 763104351884.dkr.ecr.$region.amazonaws.com/pytorch-inference:2.0.0-cpu-py310-ubuntu20.04-sagemaker
+
+RUN pip uninstall -y sagemaker_pytorch_inference
+
+COPY dist/sagemaker_pytorch_inference-*.tar.gz /sagemaker_pytorch_inference.tar.gz
+RUN pip install --upgrade --no-cache-dir /sagemaker_pytorch_inference.tar.gz && \
```

Comment:
Nit: The COPY can be done prior to the RUN commands, and the RUN commands can be combined. Marginally makes the build faster, reducing the number of layers. In this case, it's trivial.

Reply:
Acknowledged. I will send out a new commit with this change.